accounts-db/src/accounts_index/in_mem_accounts_index.rs

```diff
@@ -966,7 +966,7 @@ impl<T: IndexValue, U: DiskIndexValue + From<T> + Into<T>> InMemAccountsIndex<T,
                 continue;
             }
 
-            possible_evictions.insert(0, *k, Arc::clone(v));
+            possible_evictions.insert(0, *k);
```

Comment:
my current idea is to put `dirty` as extra value and use that for deciding to obtain the entry under map lock and clear dirty, write to disk

Reply:
yeah. it is a great idea.